supervisor/docker/addon.py

```diff
@@ -128,7 +125,6 @@ def environment(self) -> dict[str, str | None]:
             **addon_env,
             ENV_TIME: self.sys_timezone,
             ENV_TOKEN: self.addon.supervisor_token,
-            ENV_TOKEN_HASSIO: self.addon.supervisor_token,
```

Comment:
So this seems like a pretty big breaking change for addon devs, references to `HASSIO_TOKEN` will no longer work anymore right? Was there an announcement that prepared them for this? And should def include this in the upcoming post you mentioned @pvizeli

Reply:
This was announced in 2020 <https://developers.home-assistant.io/blog/2020/10/29/supervisor-update/#api-changes>
And again last year <https://developers.home-assistant.io/blog/page/2/#deprecated-api-endpoints>